packages/components/src/Banner/Banner.module.css

```diff
@@ -1,16 +1,38 @@
 .banner {
+  --banner-border: var(--color-border);
```

Comment:
Not actually using this yet but want the ability to toggle on and off quickly for demo purposes